ci/pod/docker-compose.yml

```diff
@@ -355,10 +355,34 @@ services:
     networks:
       apisix_net:
 
+  namesrv:
```

Comment:
Better to add a rocketmq prefix for the service, because this file is a common file.

Reply:
@leslie-tsang thanks, I found another way